tasks/install_and_run_ps80.yml

```diff
@@ -0,0 +1,32 @@
+# This task enables Percona-Server-80 (ps80) main repository, installs and starts percona-server.
+
+# Enable ps80 main repo and install ps80
+  - name: include tasks for enabling ps80 main repo
+    include_tasks: ../tasks/enable_ps8_main_repo.yml
+
+  - name: include tasks for installing ps80 packages
+    include_tasks: ../tasks/install_ps80.yml
+
+# Start mysql service
+  - name: start mysql service on Debian
+    service:
+      name: mysql
+      state: started
+    when: ansible_os_family == "Debian"
+
+  - name: start mysql service on CentOS
+    systemd:
+      name: mysql
+      state: started
+    when: ansible_os_family == "RedHat"
+
+# On Centos ps is started with random root password. Change ps root password and create cnf with new client password on centos.
+  - name: set root password on CentOS
```

Comment:
Thanks a lot for this, I would rather use these in pmm2-client package testing, I was handling this part by myself, btw what's the password for ps 8.x on Debian?

Reply:
discussed in Slack